File: app/src/main/java/social/entourage/android/authentication/login/register/OnRegisterUserListener.java
package social.entourage.android.authentication.login.register;

/**
 * Created by mihaiionescu on 17/06/16.
 */
public interface OnRegisterUserListener {

    void registerShowSignIn();
    boolean registerStart();
    void registerSavePhoneNumber(String phoneNumber);
    void registerCheckCode(String smsCode);
    void registerResendCode();
    void registerClosePop(Boolean isShowLogin);  //Hack en attendant la nouvelle version de l'onboarding
}

File: app/src/main/java/social/entourage/android/authentication/login/register/OnRegisterUserListener.kt
package social.entourage.android.authentication.login.register

/**
 * Created by mihaiionescu on 17/06/16.
 */
interface OnRegisterUserListener {
    fun registerShowSignIn()
    fun registerStart(): Boolean
    fun registerSavePhoneNumber(phoneNumber: String)
    fun registerCheckCode(smsCode: String)
    fun registerResendCode()
    fun registerClosePop(isShowLogin: Boolean) //Hack en attendant la nouvelle version de l'onboarding
}